src/System.AppContext/src/System/AppContext.cs

```diff
@@ -44,6 +45,19 @@ public static object GetData(string name)
             return null;
         }
 
+        public static event UnhandledExceptionEventHandler UnhandledException
```

Comment:
OK we should just fold it into the System.Runtime contract and S.P.CoreLib implementation.